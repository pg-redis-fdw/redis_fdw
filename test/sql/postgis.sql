-- Testcase 001
CREATE EXTENSION postgis;
-- Testcase 002
CREATE EXTENSION hstore;
-- Testcase 003
CREATE EXTENSION redis_fdw;

-- Testcase 004
create server localredis foreign data wrapper redis_fdw;
-- Testcase 005
create user mapping for public server localredis;

-- Testcase 006
-- \! redis-cli < test/redis_gis_ini


-- Testcase 098
-- all done, so now blow everything in the db away again
\! redis-cli < test/redis_clean

-- Testcase 099
DROP EXTENSION redis_fdw CASCADE;
-- Testcase 100
DROP EXTENSION hstore CASCADE;
-- Testcase 101
DROP EXTENSION postgis CASCADE;
