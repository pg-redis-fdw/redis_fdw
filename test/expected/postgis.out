-- Testcase 001
CREATE EXTENSION postgis;
ERROR:  extension "postgis" is not available
HINT:  The extension must first be installed on the system where PostgreSQL is running.
-- Testcase 002
CREATE EXTENSION hstore;
-- Testcase 003
CREATE EXTENSION redis_fdw;
-- Testcase 004
create server localredis foreign data wrapper redis_fdw;
-- Testcase 005
create user mapping for public server localredis;
-- Testcase 006
-- \! redis-cli < test/redis_gis_ini
-- Testcase 098
-- all done, so now blow everything in the db away again
\! redis-cli < test/redis_clean
OK
OK
-- Testcase 099
DROP EXTENSION redis_fdw CASCADE;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to server localredis
drop cascades to user mapping for public on server localredis
-- Testcase 100
DROP EXTENSION hstore CASCADE;
-- Testcase 101
DROP EXTENSION postgis CASCADE;
ERROR:  extension "postgis" does not exist
